backend/src/plugins/kube.ts

```diff
@@ -20,10 +20,16 @@ const currentContext = kc.getCurrentContext();
 const customObjectsApi = kc.makeApiClient(k8s.CustomObjectsApi);
 const coreV1Api = kc.makeApiClient(k8s.CoreV1Api);
 const batchV1Api = kc.makeApiClient(k8s.BatchV1Api);
-const batchV1beta1Api = kc.makeApiClient(k8s.BatchV1beta1Api);
+const batchV1beta1Api = kc.makeApiClient(k8s.BatchV1Api);
```

Comment:
I think we can just remove this line as the API has been removed from k8s. It doesn't make sense to have 2 x BatchV1Api clients, especially if you're calling one the V1beta1Api. 

Once you remove this, just remove the other calls to it... I'll add inline comments where that needs to be done.
```suggestion
```